### PR TITLE
solution hw-34

### PR DESCRIPTION
**Pull request by RomanHreits**

### CODE REVIEW

**Review by aleksandr-hrankin — APPROVED** [review]

**Comment by aleksandr-hrankin** [comment]
Good job!

File: src/main/java/com/internet/shop/dao/jdbc/UserDaoJdbc.java [hunk 1]
```diff
@@ -135,14 +135,16 @@ private void setValues(PreparedStatement statement, User user) throws SQLExcepti
         statement.setString(1, user.getName());
         statement.setString(2, user.getLogin());
         statement.setString(3, user.getPassword());
+        statement.setBytes(4, user.getSalt());
     }
 
     private User conversionToUser(ResultSet resultSet) throws SQLException {
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/internet/shop/dao/jdbc/UserDaoJdbc.java]:
```suggestion
    private User convertToUser(ResultSet resultSet) throws SQLException {
```

File: src/main/java/com/internet/shop/model/User.java [hunk 2]
```diff
@@ -28,6 +29,22 @@ public User(Long id, String name, String login, String password) {
...
     }
 
+    public User(Long id, String name, String login, String password, byte[] salt) {
+        this.id = id;
+        this.name = name;
+        this.login = login;
+        this.password = password;
+        this.salt = salt;
+    }
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/internet/shop/model/User.java:33]: image

Comment (dmytro-hryhoruk) [reply src/main/java/com/internet/shop/model/User.java:33]: make the same with your 4 constructors

File: src/main/java/com/internet/shop/dao/jdbc/UserDaoJdbc.java (hunk 1 above)
Comment (RomanHreits) [reply src/main/java/com/internet/shop/dao/jdbc/UserDaoJdbc.java]: done

File: src/main/java/com/internet/shop/model/User.java (hunk 2 above)
Comment (RomanHreits) [reply src/main/java/com/internet/shop/model/User.java:33]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**